server/handle_map_layer_zxy.go

```diff
@@ -158,10 +159,15 @@ func (req HandleMapLayerZXY) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	encodeCtx := context.WithValue(r.Context(), observability.ObserveVarMapName, m.Name)
 	pbyte, err := m.Encode(encodeCtx, tile)
+
 	if err != nil {
-		switch err {
-		case context.Canceled:
+		switch {
+		case errors.Is(err, context.Canceled):
 			// TODO: add debug logs
+			// do nothing
+			return
+		case strings.Contains(err.Error(), "operation was canceled"):
+			// do nothing
```

Comment:
We are trying to get away from this if possible.

Reply:
@gdey yeah, I know, but we can't in this instance. I left a code comment around this situation in `map.go`:

```go
// the underlying net.Dial function is not properly reporting
// context.Cancel errors. Becuase of this, a string check on the error is performed.
// there's an open issue for this and it appears it will be fixed eventually
// but for now we have this check to avoid unnecessary logs
// https://github.com/golang/go/issues/36208
case strings.Contains(err.Error(), "operation was canceled"):
```